api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/QualityImprovement2020DataSet.java

```diff
@@ -2334,6 +2334,26 @@ public DataSetDefinition constructQualityImprovement2020DataSet() {
             "startDate=${startDate},endDate=${endDate},revisionEndDate=${revisionEndDate},location=${location}"),
         "");
 
+    // MQ indicators category 10 denominator
+    dataSetDefinition.addColumn(
+        "MQ10DEN1A",
```

Comment:
Any special for such naming? why NOT just **MQ10DEN1** we have many more other indicators in the same range that need introduction of that, the other that follow is **MQ10DEN1C**, are we going to reuse this cohort definitions for other upcoming indicators?

Reply:
if you look at the template there, we have other levels for this category